### DISCUSSION TITLE
[BUG] “off” triggers intent “name: HassTurnOff” which turns off all lights

### DISCUSSION THREAD

**Discussion by bcutter** [OHF-Voice/intents#2360] [_General]
Please see https://community.home-assistant.io/t/off-triggers-intent-name-hassturnoff-which-turns-off-all-lights/752442 for details.

Last post is https://community.home-assistant.io/t/off-triggers-intent-name-hassturnoff-which-turns-off-all-lights/752442/10.

**Comment by bcutter** [comment]
Is this maybe due to the fact that "light" domain is not excluded AND satellite (well, it's a browser...) has no area awareness like in https://www.home-assistant.io/blog/2023/12/13/year-of-the-voice-chapter-5/#area-awareness ?

**Comment by tetele** [comment]
Converted into issue #2379
